### PR TITLE
optimise homepage FETCH_CAMPAIGNS_AND_FEATURED_PROJECTS

### PR DESCRIPTION
**Pull request by RamRamez**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new GraphQL query to retrieve both featured projects and campaigns for users based on their connected wallet.
	- Enhanced data retrieval capabilities for the homepage by updating the query structure.

- **Bug Fixes**
	- Removed outdated fields from existing queries to streamline data responses.

- **Refactor**
	- Updated data fetching logic in the homepage component to align with new backend requirements.
	- Simplified the mapping function in the LatestUpdatesBlock component for improved code readability.

- **Style**
	- Modified the styling of the Title component to allow full text display without truncation.
	- Adjusted the latest updates display to reduce the number of updates fetched from 50 to 10.

### CODE REVIEW

**Review by kkatusic — APPROVED** [review]
LGTM ;) thx @RamRamez

File: src/apollo/gql/gqlHomePage.ts [hunk 1]
```diff
@@ -66,7 +90,6 @@ export const FETCH_HOMEPAGE_DATA = gql`
 					image
 				}
 			}
-			count
```
Comment (ae2079) [inline src/apollo/gql/gqlHomePage.ts:69]: By removing this field, does it not create inconsistency between schemas?

Comment (RamRamez) [reply src/apollo/gql/gqlHomePage.ts:69]: In GraphQl you can choose which fields to fetch, and you can choose not to fetch the count

Comment (ae2079) [reply src/apollo/gql/gqlHomePage.ts:69]: Oh, I thought this is the server-side, yes that is correct

**Review by ae2079 — APPROVED** [review]
Thanks @RamRamez 
LGTM

**Review by Meriem-BM — APPROVED** [review]

**Pull request merged**